internal/apiserver/server.go

```diff
@@ -94,28 +96,13 @@ func (s *Server) GetUser(w http.ResponseWriter, r *http.Request) {
 
 func (s *Server) GetUsers(w http.ResponseWriter, _ *http.Request) {
 	log.Println("GET /user/")
-	grpcResp, err := s.grpcServer.GetUsers(context.Background(), &api.NoneObject{})
+	resp, err := s.grpcServer.GetUsers()
```

Comment:
request has a `context` that could be provided to `s.grpcServer.GetUsers` as an argument